src/server/tls.rs

```diff
@@ -56,26 +56,26 @@ impl TryFrom<FtpsConfig> for tokio_rustls::TlsAcceptor {
     }
 }
 
-pub fn new_config<P: AsRef<Path>>(certs_file: P, key_file: P) -> Arc<rustls::ServerConfig> {
-    let certs: Vec<Certificate> = load_certs(certs_file);
-    let privkey: PrivateKey = load_private_key(key_file);
+pub fn new_config<P: AsRef<Path>>(certs_file: P, key_file: P) -> std::io::Result<Arc<rustls::ServerConfig>> {
```

Comment:
suggestion(non-blocking): I prefer `use rustls::ServerConfig`, and I always do them, so looks like we have to decide if we want it consistent and if so, then in what style? Happens multiple places. As I learned it just now `std::io::Result` is better not to `use ...`. https://doc.rust-lang.org/std/io/type.Result.html so keep that one.

Reply:
I agree and fixed it. Kept `std::io::Result`